test/parallel/test-quic-client-server.js

```diff
@@ -30,8 +30,11 @@ const filedata = fs.readFileSync(__filename, { encoding: 'utf8' });
 
```

Comment:
on Line 255: is `req.authenicated` a typo ?

Reply:
Yep! sure is.